### DISCUSSION TITLE
firmware versions bolt, tag, and cf2

### DISCUSSION THREAD

**Discussion by ddwy** [bitcraze/discussions#383] [Q&A]
What do the three firmware versions bolt, tag, and cf2 represent?

**Comment by krichardsson** [comment]
The three versions are for the the three platforms: 
A [Crazyflie Bolt](https://www.bitcraze.io/products/crazyflie-bolt-1-1/) should be flashed with the bolt flavor
A [Roadrunner](https://www.bitcraze.io/products/roadrunner/) should be flashed with the tag flavor
A [Crazyflie 2.0 or 2.1](https://www.bitcraze.io/products/crazyflie-2-1/) should be flashed with the cf2 variation

There will also be a Flapper variation in the next release for the [Flapper](https://store.bitcraze.io/products/flapper-nimble-starter-kit?_pos=1&_sid=0a5e1ae24&_ss=r&variant=39813125668951)

It is important to pick the right one as device will not start with the wrong one.
